### DISCUSSION TITLE
[SOLVED] Inicio de Sesión me pide conexión a internet

### DISCUSSION THREAD

**Discussion by ValeMP** [Cotalker/documentation#79] [4. Bug Reports] [answered]
Hola!

Intento ingresar con mi usuario a una compañía que tengo en local, pero me aparece el siguiente pop-up:

![Screen Shot 2021-04-15 at 11 55 40](https://user-images.githubusercontent.com/42095617/114900093-8769ea80-9de1-11eb-8daf-5c4c149c4f53.png)

No entiendo bien por qué me solicita una conexión a internet siendo que estoy con internet. Alguna sugerencia de por qué se debe esta advertencia?

**Comment by guillevalin** [comment, marked as answer]
@ValeMP tienes que cambiar tu infraestructura de desarrollo local, hace unos dias migramos de minikube a k3d.

**Reply by ValeMP** [reply]
okis super

**Reply by acedward** [reply]
Me pasa lo mismo con k3d

**Reply by acedward** [reply]
Lo resolví en mi caso a través de agregar la configuración local en /etc/hosts
